### DISCUSSION TITLE
Add support for tasks plugin

### DISCUSSION THREAD

**Discussion by viperbbb** [davish/obsidian-full-calendar#506] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

Add support for tasks, give one more choice that this calendar can show the task lish from "tasks" plugin. although they have its own calendar, I feel too small, and the look not so good.

### ✔️ Solution

Add support for "tasks plugin"

**Comment by mkirkland4874** [comment]
I like this idea to be able to show task plugin tasks in Calendar (on a read-only basis) but I think it should be based on a custom task query (so users can configure what shows up in calendar).

I left a comment here with thoughts on a related thread.

https://github.com/davish/obsidian-full-calendar/discussions/376#discussioncomment-7711424

**Comment by YouFoundJK** [comment]
Do check out my fork [here](https://github.com/YouFoundJK/plugin-full-calendar/releases/latest) where I have it implemented.
